lib/scope.go

```diff
@@ -130,7 +130,7 @@ func (s *scope) ParseAndResolveRefWithWorkingDir(ctx context.Context, refstr, so
 
 // ParseResolveFunc returns a function that can parse a ref, then resolve and load it
 func (s *scope) ParseResolveFunc() dsref.ParseResolveLoad {
-	return NewParseResolveLoadFunc("", s.inst.defaultResolver(), s.inst)
+	return NewParseResolveLoadFunc(s.ActiveProfile().Peername, s.inst.defaultResolver(), s.inst)
```

Comment:
Allows us to use "me" as a prefix, which I'm sure we will have some other way to proxy when we have multi tenancy, but this preserve the behavior while we convert to `Dispatch` & `Scope`.

Reply:
into it. closing #1704 will help this situation